### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yoonseok312**
모바일팀 README 수정

### CODE REVIEW

**Comment by yoonseok312** [comment]
아직 작업 중입니다.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,68 @@ TinyML 번역서의 한국 독자들을 위한 한글 소스코드 저장소를
...
+        train.py를 실행시킬 때 플래그값들을 조절해서 모델을 커스텀화 할 수 있습니다. 여기서 중요한 부분들은 다음과 같습니다.
+
+        1. `—model_architecture` 어떤 네트워크를 사용할 것인지 지정하는 플래그. CNN을 기반으로한 여러 변형형태 중 하나를 선택할 수 있다. 본 프로젝트에서는 iOS 예제에서 사용하고 있는 모델인 "conv"모델을 사용하였습니다.
+        2. `—wanted_words` bulyiya, suzy
+        3. `—data_dir` 훈련에 사용할 데이터가 저장되어 있는 디렉토리. —wanted_words에서 인식하고자 한 단어들은 —data_dir에서 지정한 경로 안에 같은 이름으로 된 디렉토리를 만들고 그 안에 훈련에 사용할 파일들을 저장해야 합니다. 훈련데이터는 16bit-wav의 형식을 따라야 합니다.
+        4. `—train_dir` 훈련결과를 저장할 디렉토리. checkpoint 파일이 여기서 지정한 경로에 저장됩니다. 
+        5. `—how_many_training_steps` epoch. 데이터를 몇 바퀴 돌려서 학습할 것인지 설정합니다.
+    - freeze
+
+        아까 뭐가 생겼다 없어졌는데 ㅋㅋㅋㅋ 착각입니다 ^0^ ㅋㅎㅎㅎㅎㅎ
```
Comment (yunho0130) [inline README.md]: Remove informal experssion

File: README.md [hunk 2]
```diff
@@ -44,6 +44,68 @@ TinyML 번역서의 한국 독자들을 위한 한글 소스코드 저장소를
...
+아이폰 앱과 워치 앱을 구동시키면 스플래시 화면 이후 메인 화면이 나타나게 됩니다. 메인 화면에서 토글을 키면 입력되는 소리가 텐서플로우 모델로 들어가 inference 과정을 거칩니다. inference 후 아이폰과 워치에 각각 인식한 단어를 전해주는 알림이 전해지고, 알림은 5초 후에 꺼지거나 사용자가 탭하면 꺼집니다. 위험한 소리의 경우, 사용자가 곧바로 119에 전화할 수도 있습니다. 또한, 설정 페이지에서 인식하고 싶은 단어와 무시하고 싶은 단어를 설정할 수 있습니다. 
+
+- iOS → Watch 간의 데이터 전송
+    - iOS: MainViewModel에서 tensorflow 모델을 구동하고, 그 결과값을 WCSession을 이용해 watch에게 넘겨줍니다.
+    - Watch: WCSession에서 받은 텍스트를 WatchEnvironment라는 Watch 전체에서 쓸 수 있는 변수로 선언하여 WatchView 화면에 나타날 수 있게 했습니다.
+- viewcontroller → viewmodel / swiftUI 리펙터링
+    - 기존 tensorflowlite-ios 어플리케이션 및 타 레퍼런스의 경우 대부분 view controller 와 UIKit 을 이용하는 방식으로 되어 있기에, 최신 UI/UX 적용을 위해 SwiftUI 와 MVVM 아키텍쳐로 코드 리펙터링을 진행하였습니다.
+- gif 올리기
+
+- Reference
```
Comment (yunho0130) [inline README.md]: This document is not completed. Please check your document before PR.

**Review by yunho0130 — CHANGES_REQUESTED** [review]
Please consider adding gif file or image to show your project.

File: README.md (hunk 2 above)
Comment (yoonseok312) [reply README.md]: Marked it as draft.

**Review by yunho0130 — APPROVED** [review]
LGTM

**Pull request merged**